std/bitmanip.d

```diff
@@ -779,47 +778,62 @@ private:
     size_t* _ptr;
     enum bitsPerSizeT = size_t.sizeof * 8;
 
-    @property size_t fullWords() const @nogc pure nothrow
+    @property size_t fullWords() const @nogc pure nothrow @safe
     {
         return _len / bitsPerSizeT;
     }
+
     // Number of bits after the last full word
-    @property size_t endBits() const @nogc pure nothrow
+    @property size_t endBits() const @nogc pure nothrow @safe
     {
         return _len % bitsPerSizeT;
     }
+
     // Bit mask to extract the bits after the last full word
-    @property size_t endMask() const @nogc pure nothrow
+    @property size_t endMask() const @nogc pure nothrow @safe
     {
         return (size_t(1) << endBits) - 1;
     }
+
     static size_t lenToDim(size_t len) @nogc pure nothrow @safe
     {
         return (len + (bitsPerSizeT-1)) / bitsPerSizeT;
     }
 
+    // Set extra bits from last word to false
+    void clearExtraBits() @nogc pure nothrow
+    {
+        if (endBits)
+            _ptr[dim - 1] &= endMask;
+    }
+
 public:
     /**********************************************
-     * Gets the amount of native words backing this $(D BitArray).
+     * Returns: Number of native words backing this $(D BitArray).
      */
     @property size_t dim() const @nogc pure nothrow @safe
     {
         return lenToDim(_len);
     }
 
     /**********************************************
-     * Gets the amount of bits in the $(D BitArray).
+     * Returns: Number of bits in this $(D BitArray).
```

Comment:
I agree it is as readable as "corrected" version, I've just followed the instructions from https://dlang.org/dstyle.html which states that all functions should have at least `Params` and `Returns` sections and thought I update all the methods for consistency. Should I revert the older version?